dbaas/maintenance/managers.py

```diff
@@ -17,3 +17,14 @@ def need_retry(self, **kwargs):
             return last_task
 
         return False
+
+    @property
+    def last_available_retry(self):
+        maintenance = self.order_by("can_do_retry", "pk").last()
```

Comment:
You must put can_do_retry=1. am i right ? Because that will be return can_do_retry=0 too

Reply:
No, we are just making sure that the task we are retrying is the task that can be retried.
Sometimes the task without retry available are used to give to the user a nicer answer